weatherApp/Features/WeatherList/UI/HomeViewController.swift

```diff
@@ -12,22 +12,25 @@ class HomeViewController: UIViewController {
     
     private let rowHeight: CGFloat = 100
     private var refreshControl: UIRefreshControl!
-    private var currentWeatherList: [CurrentWeather] = []
-    private var numberOfCurrentWeather: Int {
-        currentWeatherList.count
-    }
+    private var currentWeatherListPresenter: CurrentWeatherListPresenter!
     
     @IBOutlet private weak var tableView: UITableView!
     @IBOutlet weak var tableViewHeightConstraint: NSLayoutConstraint!
     
+    convenience init(currentWeatherListViewModel: CurrentWeatherListPresenter) {
+        self.init()
+
+        self.currentWeatherListPresenter = currentWeatherListViewModel
```

Comment:
You can rename currentWeatherListViewModel to currentWeatherListPresenter and write this as:
```
self.currentWeatherListPresenter = currentWeatherListPresenter
```